index.html

```diff
@@ -131,7 +129,6 @@
 			function rettich_motion_draw_groups(lat_lng_startidx_arr=-1,lat_lng_endidx=-1) {
 				// draw all_all starting with the corresponding index. So index_set should be an array of indices with the same size as the group corresponding to CURRENT_GROUP
```

Comment:
```suggestion
			function rettich_motion_draw_groups(lat_lng_startidx_arr=-1,lat_lng_endidx=-1) {
				// draw all_all starting with the corresponding index. So index_set should be an array of indices with the same size as the group corresponding to CURRENT_GROUP
				isPaused = False
```

Reply:
hab das jetzt in [d680374](https://github.com/ReTtiChDieOrganisation/ReTtiChDasRepo/pull/95/commits/d6803749cd34e3e3db0f766095630bc67d097c1e) gefixt aber im prinzip so wie du das vorgeschlagen hast